recipes/mfem/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mfem" %}
+{% set version = "4.2" %}
+
+{% set mpi = mpi or 'mpich' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mfem/mfem/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 15ff770bb97565b085f0b8b91df60b529f427c134d1d3fe7f700aca9d3209884
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - {{ mpi }}
```

Comment:
I think this doesn't match with the docs: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mpich#message-passing-interface-mpi

Reply:
Yes, but I think it is required for cross compilation, which I'm planning to add when this becomes a feedstock. I can remove it for now if you prefer.

Reply:
It's still wrong for cross compilation. See https://github.com/conda-forge/mpi4py-feedstock/blob/master/recipe/meta.yaml#L31 for an example

Reply:
ok, fixed.